docs/reference/QueryClient.md

```diff
@@ -409,6 +409,7 @@ await queryClient.cancelQueries({ queryKey: ['posts'], exact: true })
 **Options**
 
 - `filters?: QueryFilters`: [Query Filters](../../framework/react/guides/filters.md#query-filters)
+- `cancelOptions?: CancelOptions`: [Cancel Options](../../framework/react/guides/filters.md#cancel-options)
```

Comment:
```suggestion
- `cancelOptions?: CancelOptions`: [Cancel Options](../../framework/react/guides/query-cancellation#XXX)
```

I think this would be a better place for `CancelOptions`. because `CancelOptions` is not about `filters`